piker/_daemon.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
Add the license header.

We're being pedantic about this since the sw is likely to have a lot of attention in the future.